config/custom-environment-variables.yaml

```
@@ -385,6 +385,10 @@ multiBuildCluster:
   # Enabled multi build cluster feature or not
   enabled: MULTI_BUILD_CLUSTER_ENABLED
 
+unzipArtifacts:
+  # Enabled unzip a zip artifact feature or not
```

should be same word with `/bin/server`
```suggestion
  # Enabled unzip artifacts feature or not
```